ckan/cli/views.py

```diff
@@ -307,6 +322,7 @@ def _add_default_filters(search_data_dict, view_types):
             u"recline_grid_view",
             u"recline_graph_view",
             u"recline_map_view",
+            u"datatablesplus_view",
```

Comment:
What is the `datatablesplus_view` ? Shouldn't this be `datatables_view`?

Reply:
ups, i tell the secret im working on :) 
thanks i missed it while testing